src/Component/Symfony/Bundle/Routing/Loader/CrudRoutesAttributesLoader.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Symfony\Bundle\Routing\Loader;
+
+use Sylius\Component\Resource\Annotation\SyliusCrudRoutes;
+use Sylius\Component\Resource\Reflection\ClassReflection;
+use Sylius\Component\Resource\Symfony\Routing\Loader\ResourceLoader;
+use Symfony\Bundle\FrameworkBundle\Routing\RouteLoaderInterface;
+use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\Yaml\Yaml;
+
+final class CrudRoutesAttributesLoader implements RouteLoaderInterface
+{
+    private array $mapping;
+
+    private ResourceLoader $resourceLoader;
+
+    public function __construct(
+        array $mapping,
```

Comment:
Would be cool to use to property promotion

```suggestion
        private array $mapping,
```

Reply:
indeed, but definitively in another PR using Rector :dancers:

Reply:
Extracted to https://github.com/Sylius/SyliusResourceBundle/issues/752